### PR TITLE
Adding stringify functionality

### PR DESCRIPTION
**Pull request by JeremyFarandBelanger**
- Adding the stringify function
- Fix issue when parsing SJSON containing floating number
- Updated the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,104 +12,44 @@ npm install --save simplified-json
 
 ## Usage
 
-`sample.sjson`
```
Comment (adskdave) [inline README.md:15]: I'm not sure it's a bad idea to show an example sjson file for those who don't know what it looks like.  Perhaps we could trim it down or move it down, but are you sure we should just delete it?

**Comment by adskdave** [comment]
@adskjohn i love this update, adds stringify and moves parsing to a function, but I believe we should take this

File: README.md (hunk 1 above)
Comment (JeremyFarandBelanger) [reply README.md:15]: Yes I could add some small sample.  I just felt like the actual code used to get lost in the big samples.

**Comment by JeremyFarandBelanger** [comment]
It was already there :)

**Comment by adskjohn** [comment]
Merging, I'll bump the version and put the new version up on npm soon.

**Pull request merged**